pkg/resource/deploy/step.go

```diff
@@ -1174,22 +1174,25 @@ func (s *RefreshStep) Skip() {
 }
 
 type ImportStep struct {
-	deployment    *Deployment                    // the current deployment.
-	reg           RegisterResourceEvent          // the registration intent to convey a URN back to.
-	original      *resource.State                // the original resource, if this is an import-replace.
-	old           *resource.State                // the state of the resource fetched from the provider.
-	new           *resource.State                // the newly computed state of the resource after importing.
-	replacing     bool                           // true if we are replacing a Pulumi-managed resource.
-	planned       bool                           // true if this import is from an import deployment.
-	diffs         []resource.PropertyKey         // any keys that differed between the user's program and the actual state.
-	detailedDiff  map[string]plugin.PropertyDiff // the structured property diff.
-	ignoreChanges []string                       // a list of property paths to ignore when updating.
-	randomSeed    []byte                         // the random seed to use for Check.
-	provider      plugin.Provider                // the optional provider to use.
+	deployment    *Deployment           // the current deployment.
+	reg           RegisterResourceEvent // the registration intent to convey a URN back to.
+	original      *resource.State       // the original resource, if this is an import-replace.
+	old           *resource.State       // the state of the resource fetched from the provider.
+	new           *resource.State       // the newly computed state of the resource after importing.
+	replacing     bool                  // true if we are replacing a Pulumi-managed resource.
+	ignoreChanges []string              // a list of property paths to ignore when updating.
+	randomSeed    []byte                // the random seed to use for Check.
+	provider      plugin.Provider       // the optional provider to use.
+
+	// true if this import is from an import deployment, i.e. the `pulumi import` command.
```

Comment:
I always forgot what `planned` meant, so changed the comment here to make it easier to recall for me and others in the future. `planned` == `pulumi import` command.